docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,117 @@
+//!
+//! 1. A template
+//! 2. An Omni Node
+//!
+//! While we will not do it here, it should be a great learning step to try and integrate the
+//! runtime built in this guide thus far into the [`minimal_template_runtime`], and then run it with
+//! `minimal_template_node`. This will be the former approach.o.
+//!
+//! In the next section, we will use the latter approach for simplicity. [`your_first_runtime`] uses
+//! has no consensus related code, and therefore can only be executed with a node that uses
+//! [`sc_consensus_manual_seal`], namely `minima-omni-node`.
+//!
+//! ## Running Your First (omni) Node.
+//!
+//! For the rest of this tutorial, we will only focus on running the runtime of the previous guide
+//! using the `minimal-omni-node`. This section will be updated with more info once the omni-node
+//! feature is fully merged.
+//!
+//! The process of running a runtime with an omni-node is quite trivial:
+//!
+//! * compile the runtime (with `--release`)
+//! * pass it to the omni-node as the `--runtime`.
+//! * Note that we use the [`--tmp`](sc_cli::RunCmd::tmp) flag which spins up a new temporary
+//!   database each time.
+//! * We enable all `runtime` logs, which helps with the visibility of what is happening in the
+//!   runtime. If you now add `frame::log::info!(target: "runtime", ...)` logs in your runtime, you
+//!   will see them in the output.
+//!
+//! ```ignore
+//! $ cargo build --release -p polkadot-sdk-docs-packages-guides-first-runtime
+//! # assuming the `minimal-omni-node` binary is available from compiling it from
+//! # https://github.com/paritytech/polkadot-sdk/pull/3597
+//! $ ./minimal-omni-node\
+//!     --tmp \
+//!     --runtime ./target/release/wbuild/polkadot-sdk-docs-packages-guides-first-runtime/polkadot_sdk_docs_packages_guides_first_runtime.wasm \
+//!     -l runtime=debug
+//! ```
+//!
+//! Or the equivalent from the tests of this crate:
+#![doc = docify::embed!("./src/guides/your_first_node.rs", node_process)]
+//!
+//! The only detail that we will cover for now is that to enable an easy way for the runtime to
+//! generate some initial state, as it greatly makes the development process easier. For the time
+//! being, please pay attention to the implementation of [`build_config`] in the corresponding
+//! `Runtime`. For now, we always populate the chian with some funds in all of the accounts named in
+//! [`sp_keyring`]. UIs such as Polkadot-JS-Apps are pre-configured to detect and show the balance
+//! of these accounts.
+//!
+//! Once launched, you can navigate to <https://polkadot.js.org/apps/>, connect it to
+//! `ws://localhost:9944` and see the chain in action. Please see the corresponding [documentation
+//! of Polkadot-JS-Apps](https://polkadot.js.org/docs) for more information.
+//!
+//! [`build_config`]: polkadot_sdk_docs_packages_guides_first_runtime::Runtime#method.build_config
+
+#[test]
+#[ignore = "will not work until we have good omni-nodes in this repo; wait for #3597"]
```

Comment:
```suggestion
#[cfg(feature = "will not work until we have good omni-nodes in this repo; wait for #3597")]
```
I think CI runs with `--ignored`.